lib/licensee/license.rb

```diff
@@ -38,13 +50,23 @@ def find(key, options = {})
       alias [] find
       alias find_by_key find
 
-      def license_dir
-        dir = ::File.dirname(__FILE__)
-        ::File.expand_path '../../vendor/choosealicense.com/_licenses', dir
+      def license_dirs
+        return @license_dirs
+      end
+
+      # A user customized lookup array of directories for license files. If the user wants 
+      # to include the default, they should make one of the entries be default_license_dir()
+      def license_dirs=(dirs)
+        @license_dirs=dirs
       end
 
+      # Returns all the license files across the various license directories
       def license_files
-        @license_files ||= Dir.glob("#{license_dir}/*.txt")
+        files=[]
+        @license_dirs.each do |dir|
+          files.concat(Dir.glob("#{dir}/*.txt"))
+        end
+        @license_files ||= files
```

Comment:
This doesn't properly memoize the method because the `Dir.glob` logic will always run.  How about

```ruby
@license_files ||= license_dirs.map { |d| Dir.glob("#{d/*.txt") }.flatten
```